assets/stylesheets/components/_registration.scss

```diff
@@ -0,0 +1,6 @@
+.registration {
```

Comment:
Avoid single word component to avoid class name conflicts:

> Class names must be composed of at least two words to make each element class name specific and more descriptive.
> [Compass](https://nimblehq.co/compass/development/code-conventions/css/#class-names)